src/Apps/Auction/Components/AuctionActiveBids.tsx

```diff
@@ -218,11 +226,15 @@ const BidButton: React.FC<
   const { router } = useRouter()
   const { tracking } = useAuctionTracking()
 
-  if (!lotStanding) {
-    return null
-  }
+  if (!lotStanding) return null
+
+  const { saleArtwork } = lotStanding
+  if (!saleArtwork) return null
 
-  if (!!lotStanding.saleArtwork?.endedAt) {
+  const { sale } = saleArtwork
+  if (!sale) return null
+
+  if (!!saleArtwork.endedAt) {
```

Comment:
I think this line needs to go after `if (!!sale.isLiveOpen)` - a live open sale will not be ended.